config/amp.go

```diff
@@ -4,7 +4,7 @@ const (
 	// AmplifierDefaultEndpoint is the default amplifier endpoint
 	AmplifierDefaultEndpoint = "amplifier:50101"
 
-	// EtcdDefaultEndpoints is the default etcd endpoint
+	// EtcdDefaultEndpoint is the default etcd endpoint
```

Comment:
Fwiw, this was originally pluralized intentionally because etcd takes a comma-separated list for default endpoints. Not a big deal though.